packages/cli/src/cmds/stats/stats.ts

```diff
@@ -0,0 +1,319 @@
+import runCommand from '../runCommand';
+import yargs from 'yargs';
+import { chdir } from 'process';
+const fs = require('fs');
+import UI from '../userInteraction';
+import Telemetry from '../../telemetry';
+import { listAppMapFiles, verbose } from '../../utils';
+
+export const command = 'stats [directory]';
+export const describe =
+  'Show some statistics about events in scenarios read from AppMap files';
+const LIMIT_DEFAULT = 5;
+const MINIMUM_APPMAP_SIZE = (1024 * 1024) / 2;
+
+export const builder = (args: yargs.Argv) => {
+  args.option('directory', {
+    describe: 'Working directory for the command.',
+    type: 'string',
+    alias: 'd',
+  });
+
+  args.option('json', {
+    describe: 'Format results as JSON.',
+    type: 'boolean',
+    alias: 'j',
+  });
+
+  args.option('limit', {
+    describe:
+      'limit the number of methods displayed (default ' + LIMIT_DEFAULT + ').',
+    type: 'number',
+    alias: 'l',
+  });
+
+  return args.strict();
+};
+
+export async function handler(argv: any) {
+  verbose(argv.verbose);
+
+  const commandFn = async () => {
+    const { directory, json, limit } = argv;
+    if (directory) {
+      if (verbose()) console.log(`Using working directory ${directory}`);
+      chdir(directory);
+    }
+    let limitToUse = limit;
+    if (!limitToUse) {
+      limitToUse = LIMIT_DEFAULT;
+    }
+
+    let directoryToUse = directory;
+    if (!directoryToUse) directoryToUse = '.';
+
+    async function calculateAppMapSizes(appMapDir: string) {
+      const appMapSizes = {};
+
+      // This function is too verbose to be useful in this context.
+      const v = verbose();
+      verbose(false);
+      await listAppMapFiles(appMapDir, (fileName: string) => {
+        const stats = fs.statSync(fileName);
+        appMapSizes[fileName] = {
+          size: stats.size,
+        };
+      });
+      verbose(v);
+
+      return appMapSizes;
+    }
+
+    async function sortAppMapSizes(appMapSizes) {
+      let appMapSizesArray: { name: string; size: number }[] = [];
+
+      for (const key in appMapSizes) {
+        appMapSizesArray.push({
+          name: key,
+          size: appMapSizes[key].size,
+        });
+      }
+
+      return appMapSizesArray.sort((a, b) => b.size - a.size);
+    }
+
+    function sizeInMB(size) {
+      return Number((size / 1000 / 1000).toFixed(1));
+    }
+
+    async function calculateExecutionTimes(appMapDir) {
+      let functionExecutionTimes = {};
+
+      // This function is too verbose to be useful in this context.
+      const v = verbose();
+      verbose(false);
+      await listAppMapFiles(appMapDir, (fileName: string) => {
+        const file = fs.readFileSync(fileName, 'utf-8');
+        let appmap = JSON.parse(file.toString());
+        functionExecutionTimes[fileName] = {
+          functions: {},
+        };
+
+        appmap.events.forEach((event) => {
+          if (event.event == 'call') {
```

Comment:
Use event.isCall

Reply:
`event.isCall` doesn't match.
`event.isCall()` doesn't gives the error
```
An error occurred: TypeError: event.isCall is not a function
```

I think it needs to read the event differently. Right now it's a datastructure read from `JSON.parse`